packages/components/src/Typeahead/Typeahead.snapshot.test.js

```diff
@@ -153,6 +173,37 @@ describe('Typeahead', () => {
 			// then
 			expect(wrapper).toMatchSnapshot();
 		});
+
+		it('should render typeahead selected item by id', () => {
+			// given
+			const props = {
+				id: 'my-search',
+				value: 'le title 1',
+				valueId: 'letitle1copy',
+				items: itemsObjectWithId,
+			};
+
+			// when
+			const wrapper = renderer.create(<Typeahead {...props} />).toJSON();
+
+			// then
+			expect(wrapper).toMatchSnapshot();
```

Comment:
Are we sure it is this one selected? Because we only see the title `le title 1` in the snapshot, not the associated id. 🤔

Reply:
Please avoid snapshots. I know there are lots of snapshots in this file but we need to get rid of that.
The reason is that, looking at your test, it's really hard to check if the snapshot is correct. Don't snapshot tons of html to check only if the first option is selected.
Instead, you can query element with id `react-autowhatever-my-search-section-0-item-0` and assert that aria-selected is true